app/views/sessions/_filters.html.erb

```diff
@@ -3,28 +3,87 @@
 <div class="flex justify-between w-full mb-6">
   <div class="flex items-center gap-2 overflow-x-scroll no-scrollbar">
     <%= link_to(
-      'All dates',
-      url_for(resource),
-      class: [
-        "flex min-w-fit items-center justify-center px-4 py-2 rounded-full font-medium",
-        "bg-red text-white": params[:starts_at].blank?,
-        "bg-white text-red": params[:starts_at].present?
-      ]
-    ) %>
+          'All dates',
+          url_for(resource),
+          class: [
+            "flex min-w-fit items-center justify-center px-4 py-2 rounded-full font-medium",
+            "bg-red text-white": params[:starts_at].blank?,
+            "bg-white text-red": params[:starts_at].present?
+          ]
+        ) %>
     <% current_conference.session_dates.each do |session_date| %>
       <%= link_to(
-        session_date.strftime("%a, %d"),
-        url_for([resource, starts_at: session_date]),
-        class: [
-          "flex min-w-fit items-center justify-center px-4 py-2 font-medium rounded-full",
-          "bg-red text-white": params[:starts_at] == session_date.to_s,
-          "bg-white text-red": params[:starts_at] != session_date.to_s
-        ]
-      ) %>
+            session_date.strftime("%a, %d"),
+            url_for([resource, starts_at: session_date]),
+            class: [
+              "flex min-w-fit items-center justify-center px-4 py-2 font-medium rounded-full",
+              "bg-red text-white": params[:starts_at] == session_date.to_s,
+              "bg-white text-red": params[:starts_at] != session_date.to_s
+            ]
+          ) %>
     <% end %>
   </div>
 
-  <div class="flex items-center justify-center ml-3 p-2 bg-white rounded-full min-w-[40px] min-h-[40px]">
-    <%= inline_svg_tag("icons/filters.svg", size: "18") %>
+  <div class="flex items-center justify-center ml-3 p-2 rounded-full min-w-[40px] min-h-[40px] <%= active_session_filters? ? "bg-red text-white" : "bg-white text-red" %>">
+    <div class="relative" data-controller="dropdown">
+      <button type="button" data-action="dropdown#toggle click@window->dropdown#hide" class="flex items-center justify-center p-2 rounded-full">
+        <%= inline_svg_tag("icons/filters.svg", size: "18") %>
```

Comment:
When filters are applied, the filter icon sh
<img width="188" alt="Screenshot 2024-08-15 at 3 24 40 p m" src="https://github.com/user-attachments/assets/99f445e9-47f9-415a-a4a2-8acc26e715a7">
ould turn white